tests/subsys/emds/emds_flash/src/main.c

```diff
@@ -0,0 +1,699 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <ztest.h>
+#include <errno.h>
+#include <zephyr.h>
+#include <storage/flash_map.h>
+#include <drivers/flash.h>
+#include <sys/crc.h>
+#include <dk_buttons_and_leds.h>
+#include <../subsys/emds/emds_flash.h>
+
+#if defined(CONFIG_BT)
```

Comment:
What is sense to have condition compilation in the test if there is the strictly defined configuration file? CI doesn't combine configuration options by itself.

Reply:
It makes it easy to see what happends when CONFIG_BT is removed from the prj.conf file. So this was added when it was tested with and without BT. So it is left there for convience. It is also to show that the mpsl scheduler does impact write times. So I would like to leave it, as this is a test.